sdk/python/packages/flet-cli/src/flet_cli/commands/serve.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import http.server
+import socketserver
+from pathlib import Path
+
+from flet_cli.commands.base import BaseCommand
+from rich.console import Console
+from rich.style import Style
+
+error_style = Style(color="red1", bold=True)
+console = Console(log_path=False)
+
+
+class CustomHandler(http.server.SimpleHTTPRequestHandler):
+    def __init__(self, *args, directory=None, **kwargs):
+        super().__init__(*args, directory=directory, **kwargs)
+
+    def end_headers(self):
+        self.send_header("Cross-Origin-Opener-Policy", "same-origin")
+        self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
+        self.send_header("Access-Control-Allow-Origin", "*")
+        super().end_headers()
+
+
+class Command(BaseCommand):
+    """
+    Serve static files from a directory with optional WASM headers.
+    """
+
+    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument(
+            "web_root",
+            type=str,
+            nargs="?",
+            help="directory to serve (default: ./build/web)",
+            default="build/web",
+        )
+        parser.add_argument(
+            "--port",
+            type=int,
+            default=8000,
+            help="Port to serve on (default: 8000)",
+        )
```

Comment:
I suggest we support the shortform `-p` too. Similar to what we have in `flet run`.